src/main/resources/META-INF/plugin.xml

```diff
@@ -49,6 +49,12 @@
                 <override-text place="GitMacheteContextMenu"/>
             </action>
 
+            <action id="GitMachete.SquashSelectedAction"
```

Comment:
Hmmm so what's finally the convention if it comes to putting `Branch` or not in action class names? 🤔

Reply:
(maybe the current state is something I enforced? 😅)

Reply:
I don't think we ve got a rule on this. 
Personally, I would rather drop "branch" word from the action ids (and class names too) to keep them shorter (for me the references to "current" and "selected" in scope of action ids/class names).

Reply:
wdyt @PawelLipski ?

Reply:
Hmmm I thought we've got a task for that already... anyway, pls re-open or open a new one, and/or check what was the conclusion of the old PR (maybe we decided against removing `Branch` from everywhere coz it sounded awkward/confusing)?

Reply:
let's fight [here](https://github.com/VirtusLab/git-machete-intellij-plugin/pull/1059)